assets/style.css

```diff
@@ -37,10 +42,11 @@ iframe {
 }
 
 #site-footer {
+  height: 60px;
+  position: absolute;
+  line-height: 20px;
   background-color: darkslategrey;
   max-width: 100%;
-  overflow-x: hidden;
-  position: fixed;
```

Comment:
![image](https://user-images.githubusercontent.com/16431379/91013819-86135580-e5a5-11ea-9793-25d33fcbe2ea.png)


I think we want to keep position: fixed, absolute makes it go into the middle of the page on mine when I scroll, fixed works for me though.

Reply:
If you just make the footer an absolute position this will happen, you need the html height and position to be what they are here too.